monitor/services/src/main/java/org/datacleaner/monitor/server/filesystem/IncompleteFileException.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.monitor.server.filesystem;
+
+import java.io.IOException;
+
+public class IncompleteFileException extends IOException {
```

Comment:
I can't decide if I really agree that this is an `IOException`. The IO channels might be functioning perfectly, it just took longer than expected (or program was shut down). I don't really mind though, so let's leave as-is unless others also think it's a wrong thing to inherit from.